static-site/src/pages/groups.jsx

```diff
@@ -39,11 +41,10 @@ const tableColumns = [
   },
   {
     name: "Group Name",
-    value: (dataset) => dataset.contributor,
-    url: (dataset) => dataset.contributorUrl,
-    logo: (dataset) => dataset.contributor==="Nextstrain" ?
-      <img alt="nextstrain.org" className="logo" width="24px" src={nextstrainLogoPNG}/> :
-      <MdPerson/>
+    value: (dataset) => dataset[DATASET_GROUPNAME_KEY]
+    // TODO add a link when we collect groups datasets to individual group pages
```

Comment:
Is there anything stopping us from doing this now? A link to `/groups/${dataset[DATASET_GROUPNAME_KEY]}` will work as-is (currently it'd be rendered through Auspice, after #314 it'll be rendered by Gatsby)

Reply:
Good point. I think we are about to merge #324 which will set this value so I'll leave this for now.